### DISCUSSION TITLE
Laravel Farm Plugin? How to use farm (vite alternative) with laravel

### DISCUSSION THREAD

**Discussion by gabrielrbarbosa** [laravel/framework#51959] [General]
Im trying to install https://github.com/farm-fe/farm in my project that is currently using vite+vue2, does anyone knows if it is possible or do we need to wait for a laravel-farm-plugin?

**Comment by albetnov** [comment]
Haven't tried it, but as per documentation farm support loading Vite Plugins as well. Perhaps you could put the `laravel-vite-plugin` into `vitePlugins` in farm config?

**Reply by QuentiumYT** [reply]
Sadly, it's not supported yet `Vite plugin 'laravel' is not compatible with Farm for now. Because it uses viteConfig.build['manifest'] which is not supported by Farm. Current supported keys are: outDir,sourcemap,minify,cssMinify,ssr,watch,rollupOptions,assetsDir` hopefully soon :eyes:
